app/views/coronavirus_form/results.html+govwales.erb

```diff
@@ -0,0 +1,64 @@
+<% title = results_title(result_groups(session)) %>
+
+<%# GOVWALES: :back_link addition. Lines 3-10 %>
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", {
+    text: t("govwales_breadcrumbs.start_again_text"),
+    href: t("govwales_breadcrumbs.start_again_url"),
+    start: true,
+  } %>
+<% end %>
+
+<% content_for :title do %>
+  <%= title %>
+<% end %>
+
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= title %>" />
+<% end %>
+
+<% content_for :page_header do %>
+  <%= render "govuk_publishing_components/components/title", {
+    context: t("coronavirus_form.results.header.context"),
+    title: t("coronavirus_form.results.header.title"),
+    inverse: true,
+    margin_top: 0,
+    margin_bottom: 6,
+  } %>
+  <%# GOVWALES: '.govwales-removed' addition. Lines 28-29 %>
+  <%= link_to t("coronavirus_form.results.header.start_again_text"), clear_session_path, class: "govuk-link govwales-removed", onclick: "ga('send', 'event', 'StartAgain', 'StartAgainClicked', '/results');" %>
+<% end %>
+
+<%# GOVWALES: 'govuk-width-container' addition. Lines 33 & 57 %>
+<div class="govuk-width-container">
+  <% if result_groups(session).empty? %>
+    <%= sanitize(t("coronavirus_form.results.no_results")) %>
+  <% else %>
+    <% result_groups(session).each do |group| %>
+      <%= render "components/actions-group", {
+        title: group[1][:heading],
+        subsections: group[1][:questions]
+      } %>
+    <% end %>
+  <% end %>
+
+  <%# GOVWALES: Back button addition. Lines 44 to 48 %>
+  <%= render "govuk_publishing_components/components/button", {
+    text: t("govwales_buttons.start_again_text"),
+    href: t("govwales_buttons.start_again_url"),
+    start: true
+  } %>
+
+  <%= render "components/callout", {
+    title: "Help us improve"
```

Comment:
This needs translating...

Reply:
This is the original GDS one, that we don't see. Could pick it up in a snag task?